lib/ansible/modules/packaging/os/flatpak.py

```diff
@@ -94,6 +94,11 @@
   flatpak:
     name: org.gnome.gedit
     state: absent
+
+- name: Install a list of flatpaks
+  flatpak:
+    name: ["io.atom.Atom", "org.libreoffice.LibreOffice"]
```

Comment:
For documentation, we generally prefer the list format:
```
- name: Install a list of flatpaks
  flatpak:
     name:
      - "io.atom.Atom"
      - "org.libreoffice.LibreOffice"
```

Reply:
I have now updated the example to be a YAML list.